server/interfaces.coffee

```diff
@@ -33,3 +34,15 @@ Meteor.methods
         dateUTC = match[1].replace(' ', 'T') + offset
         dateUTC
     )
+
+  queryForSuggestedArticles: (eventId) ->
+    event = grid.UserEvents.findOne(eventId)
+    console.log "Calling SPA API @ " + SPA_API_URL
+    response = HTTP.call('GET', "#{SPA_API_URL}/relatedArticles", {
+      params: { eventName: event.summary }
```

Comment:
My only concern with that approach would be the ease of overloading the database engine by supplying a very permissive query. If we are okay to allow any queries to go through this API and intend to limit the access by whitelisting IPs, I'm totally alright with that.